main.js

```diff
@@ -1,352 +1 @@
-            }
-            console.log('binch')
-            document.getElementById('error1').style.display = 'block';
-            return;
-        }
-
-        toggleModal(e);
-        gameFlow.resetGame(); // Reset Game
-        gameFlow.players = players;// Fill players object
-        for (let i = 0; i < 9; i++){ // Add event listeners to game pieces
-            gameBoard.tiles[i].addEventListener('click', gameFlow.addMark);
-        }
-    },
-
-    computerMove() {
-        console.log('entered computer move')
-        let tiles = gameBoard.availableTiles(); // Get array of available tiles
-        let choice = Math.floor(Math.random() * tiles.length); // Create random number <= length of array
-        return tiles[choice]; // return random number as tile index
-    },
-
-    addMark(e) {
-        // Buttons only function if user has selected player form
-        if (gameFlow.players.player1 === undefined) {
-            return;
-        }
-
-        //Get players
-        const player1 = gameFlow.players.player1;
-        const player2 = gameFlow.players.player2;
-
-        // Find tile index from button id
-        const index = e.target.getAttribute('id').substring(1);
-        // Ensure space is available
-        if (!gameBoard.isAvailable(index)){
-            return;
-        }
-        // Change background color
-        e.target.className = 'selectedTile';
-        // If 1-player game
-        if (player2.name === 'computer') {
-
-            // Add mark to game tile & update counter
-            gameBoard.showIcon(index, player1.mark);
-            counter.add()
-
-            // Begin checking for winner if move count >= 5
-            if (counter.count >= 5) {
-                if (gameFlow.hasWon()){
-                    gameFlow.returnWinner(player1.mark);
-                    gameFlow.resetGame();
-                    return;
-                }
-                if (counter.count === 9) {
-                    // Check for tie
-                    if (gameFlow.hasTie()) {
-                        gameFlow.resetGame();
-                        return;
-                    }
-                } 
-            }
-
-            // Get computer move
-            let computerIndex = gameFlow.computerMove();
-            gameBoard.showIcon(computerIndex, player2.mark);
-            gameBoard.tiles[computerIndex].className = 'selectedTile';
-            counter.add()
-            //Check for winner again
-            if (counter.count >= 5) {
-                if (gameFlow.hasWon()){
-                    gameFlow.returnWinner(player2.mark);
-                    gameFlow.resetGame();
-                    return;
-                }
-                if (counter.count === 9) {
-                    // Check for tie
-                    if (gameFlow.hasTie()) {
-                        gameFlow.resetGame();
-                        return;
-                    }
-                } 
-            }
-            return;
-        }
-
```

Comment:
this is fine
If it were me I would remove the counter completely
  At the end of each turn, check for win/tie and switch players